### PR TITLE
fix some linters issue

### LINKED ISSUE #198: Bug: fix some linters issue
Fix some linters issue

### PR DESCRIPTION
**Pull request by sashamelentyev**
# Description

Fix some linters issue, but not all. We can stay open this issue. Or close and create new next time

Fixes #198

### CODE REVIEW

File: pipeline/processor.go [hunk 1]
```diff
@@ -93,13 +93,13 @@ func NewProcessor(
 	return processor
 }
 
-func (p *processor) start(params *PluginDefaultParams, logger *zap.SugaredLogger) {
+func (p *processor) start(params *PluginDefaultParams, l *zap.SugaredLogger) {
```
Comment (ansakharov) [inline pipeline/processor.go:96]: l -> log

File: plugin/output/file/file_test.go [hunk 2]
```diff
@@ -85,14 +85,14 @@ func TestGetStartIdx(t *testing.T) {
 }
 
 func TestSealUpHasContent(t *testing.T) {
-	cfg := Config{
+	c := Config{
```
Comment (ansakharov) [inline plugin/output/file/file_test.go:88]: c -> config

File: fd/util.go [hunk 3]
```diff
@@ -97,7 +97,10 @@ func extractMatchMode(actionJSON *simplejson.Json) (pipeline.MatchMode, error) {
 }
 
 func extractMatchInvert(actionJSON *simplejson.Json) (bool, error) {
-	invertMatchMode := actionJSON.Get("match_invert").MustBool()
+	invertMatchMode, err := actionJSON.Get("match_invert").Bool()
```
Comment (ansakharov) [inline fd/util.go:100]: I need look deeper at this case, probably it can fail at upper level: it checks for err != nil, while err always null

**Review by ansakharov — CHANGES_REQUESTED** [review]

**Comment by ansakharov** [comment]
Your new linter failed lint:
```
prepare environment
run golangci-lint
  Running [/home/runner/golangci-lint-1.45.2-linux-amd64/golangci-lint run --out-format=github-actions --new-from-patch=/tmp/tmp-1620-fWKuP0XhAj4q/pull.patch --new=false --new-from-rev= --timeout 5m] in [] ...
  level=error msg="Running error: unknown linters: 'usestdlibvars', run 'golangci-lint help linters' to see the list of supported linters"
  
  Error: golangci-lint exit with code 3
  Ran golangci-lint in 1616ms
```

File: plugin/output/file/file_test.go (hunk 2 above)
Comment (sashamelentyev) [reply plugin/output/file/file_test.go]: Done

File: fd/util.go (hunk 3 above)
Comment (sashamelentyev) [reply fd/util.go]: I revert this case

**Comment by sashamelentyev** [comment]
> Your new linter failed lint:
> 
> ```
> prepare environment
> run golangci-lint
>   Running [/home/runner/golangci-lint-1.45.2-linux-amd64/golangci-lint run --out-format=github-actions --new-from-patch=/tmp/tmp-1620-fWKuP0XhAj4q/pull.patch --new=false --new-from-rev= --timeout 5m] in [] ...
>   level=error msg="Running error: unknown linters: 'usestdlibvars', run 'golangci-lint help linters' to see the list of supported linters"
>   
>   Error: golangci-lint exit with code 3
>   Ran golangci-lint in 1616ms
> ```

I update golangci-lint

**Comment by vadimalekseev** [comment]
@sashamelentyev, thanks for this contribution!

Can we hardcode golangci-lint version in the Makefile too?
For example:
```
.PHONY: lint
lint:
	# installation: https://golangci-lint.run/usage/install/#local-installation
	golangci-lint run --new-from-rev=${UPSTREAM_BRANCH}
```

->

```
.PHONY: lint
lint:
	go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.49.0 run --new-from-rev=${UPSTREAM_BRANCH}
```

**Comment by sashamelentyev** [comment]
@vadimalekseev done

**Comment by sashamelentyev** [comment]
@ansakharov @vadimalekseev can we merge this?

**Pull request closed without merging**